README.md

```diff
@@ -330,7 +330,42 @@ ctest -C RelWithDebInfo --parallel 8
 On Mac OSX using Xcode
 ======================
 
-#### Get the dependencies
+#### For Mac OSX 10.10 Yosemite and OS X 10.11 El Capitan
+Get **Xcode** from the App store. Open **Xcode** and *Agree* to license agreement. To *Agree* to to the license agreement, you may need to type in **Terminal**:
+```shell 
+sudo xcodebuild -license
+``` 
+
+Then, in **Terminal**, copy and paste commands below, line by line, one at a time. Be sure the output doesn't contain errors.
+```shell
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+brew install cmake swig
+brew cask install java
+git clone https://github.com/opensim-org/opensim-core.git
+mkdir opensim_dependencies_build
+cd opensim_dependencies_build
+cmake ../opensim-core/dependencies \
+      -DCMAKE_INSTALL_PREFIX="~/opensim_dependencies_install" \
+      -DCMAKE_BUILD_TYPE=RelWithDebInfo
+make -j8
+cd ..
+mkdir opensim_build
+cd opensim_build
+cmake ../opensim-core \
+      -DCMAKE_INSTALL_PREFIX="~/opensim_install" \
+      -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+      -DBUILD_PYTHON_WRAPPING=ON \
+      -DBUILD_JAVA_WRAPPING=ON \
+      -DOPENSIM_DEPENDENCIES_DIR="~/opensim_dependencies_install" \
+      -DWITH_BTK=ON
+make -j8
+ctest -j8
+```
+
+
+#### Extended Instructions for OSX
```

Comment:
Could you alter the headings in this "long" section so that each step is at the same heading level? Right now, other than "Get the dependencies", the headings in this section are at the same level as "Extended Instructions...," which could be confusing. I'm fine with either moving the rest of the headings to `#####` or moving `Extended Instructions` to `###` (and "Get the dependencies to `####`).